ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/views/dialogs/SaveHierarchiesDialog.java

```diff
@@ -0,0 +1,154 @@
+package org.iplantc.de.admin.desktop.client.ontologies.views.dialogs;
+
+import org.iplantc.de.admin.desktop.client.ontologies.OntologiesView;
+import org.iplantc.de.admin.desktop.client.ontologies.events.SaveOntologyHierarchyEvent;
+import org.iplantc.de.client.models.IsHideable;
+import org.iplantc.de.client.models.ontologies.Ontology;
+import org.iplantc.de.commons.client.validators.UrlValidator;
+import org.iplantc.de.commons.client.views.dialogs.IPlantDialog;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+import com.google.gwt.event.shared.HasHandlers;
+import com.google.inject.Inject;
+
+import com.sencha.gxt.core.client.dom.ScrollSupport;
+import com.sencha.gxt.widget.core.client.button.TextButton;
+import com.sencha.gxt.widget.core.client.container.FlowLayoutContainer;
+import com.sencha.gxt.widget.core.client.event.SelectEvent;
+import com.sencha.gxt.widget.core.client.form.FieldLabel;
+import com.sencha.gxt.widget.core.client.form.TextField;
+import com.sencha.gxt.widget.core.client.toolbar.ToolBar;
+
+import java.util.List;
+
+/**
+ * @author aramsey
+ */
+public class SaveHierarchiesDialog extends IPlantDialog implements IsHideable {
```

Comment:
`PublishOntologyDialog` and `CategorizeDialog` were small and simple enough that I didn't do a separate view or do a UI binding.  `SaveHierarchiesDialog` looks big, but it's a small dialog with just a field to type in a root IRI with buttons to either add or delete more fields if you need to add multiple root IRIs.